priv/repo/migrations/20160907190212_create_location.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Gatherto.Repo.Migrations.CreateLocation do
+  use Ecto.Migration
+
+  def change do
+    create table(:locations) do
+      add :postal_code, :string
+      add :state, :string
+      add :city, :string
+      add :latitude, :float
+      add :longitude, :float
+
+      timestamps()
```

Comment:
I don't think for this model - this all came from seed data (```priv/repo/locations.csv```). The thing I was thinking about was using this as a cross reference. This needs a lot of thinking through from a high level with use cases I have in mind. I should probably enumerate those use cases as that would help :)